notebooks/0. Complete solution.ipynb

```diff
@@ -168,8 +174,8 @@
     "car_params = {'min_height': 1.2, 'max_height': 2.4, 'min_width': 1.4, 'max_width': 2.4, 'min_length': 3.0, 'max_length': 6.0}\n",
     "\n",
     "# Car fuser using a clustering algorithm and BGT road data.\n",
-    "car_fuser = fusion.CarFuser(Labels.CAR, ahn_reader, bgt_file=bgt_road_file, grid_size=0.05,\n",
-    "                            min_component_size=5000, params=car_params)"
+    "car_fuser = fusion.CarFuser(Labels.CAR, ahn_reader, bgt_reader=bgt_road_reader, exclude_types=[\"fietspad\"],\n",
```

Comment:
Vanwaar de \ hier? exclude_types=["fietspad",]

Reply:
Dat is alleen in de diff viewer zo. Komt vanwege de dubbele aanhalingstekens.